File: 프로그래머스/unrated/181874. A 강조하기/A 강조하기.js
function solution(myString) {
  let ans = []

 for (let i = 0; i < myString.length; i++) {
  if (myString[i] == "a" || myString[i] == "A") {
    ans.push(myString[i].toUpperCase())
  } else {
    ans.push(myString[i].toLowerCase())
  }
 }
 return ans.join('')
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
class Solution {
  public String solution(String myString) {
    StringBuilder sb = new StringBuilder();

    for (String str : myString.split("")) {
      if (str.equals("a") || str.equals("A")) sb.append(str.toUpperCase());
      else sb.append(str.toLowerCase());
    }

    return sb.toString();
  }
}